modules/apps/calendar/calendar-web/src/main/resources/META-INF/resources/css/main.scss

```diff
@@ -433,6 +444,10 @@
 	}
 }
 
+.col-md-4 .calendar-portlet .popover.scheduler-event-recorder-popover {
```

Comment:
Hey @naraandrade, Is it necessary to add `col-md-4` to this selector?

Reply:
Hi @interaminense :) Yes, this was the simplest way I identified to adjust the size of the popover when the calendar is used in a narrow space (a page with 2 columns 30/70, for example).